mypy.ini

```diff
@@ -0,0 +1,5 @@
+[mypy]
+python_version = 3.7
```

Comment:
I believe we have python 3.8 dependency in our CI setup - could we change this to 3.8 instead?